server_environment/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Wingo SA
+# Copyright 2018 Camptocamp (https://www.camptocamp.com).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+{
+    "name": "server configuration environment files",
+    "version": "16.0.1.1.1",
```

Comment:
Bump module version to 16.0.1.0.0

```suggestion
    "version": "16.0.1.0.0",
```

Reply:
@FrancoMaxime can you address this comment please? The version change should belong to the migration commit indeed.

Reply:
Yes sorry,  I'm sick, I'll fix it tomorrow

Reply:
It's fine, we are not in a hurry :)
Could you cherry-pick these fixes as well https://github.com/OCA/server-env/pull/126/commits ?
In the meantime, rest well!

Reply:
@sebalix I took the initiative to apply the requested changes to move forward.